src/disk.hpp

```diff
@@ -97,24 +97,32 @@ struct FileDisk {
     explicit FileDisk(const fs::path &filename)
     {
         filename_ = filename;
-        Open(false);
+        Open();
```

Comment:
actually. here's a case where we used to say read/write. but you changed it to rely on the default, so this changed to read-only now. since there is not "read_write" flag (it's just the absence of `ReadOnly`) maybe it would make sense to either:

1. invert the flag and have a `WriteFlag` instead of a `ReadOnlyFlag`. Then the default flag could be 0.
2. Change the default flag to be 0 and review all places we call `Open()` from to ensure we pass the right flag.
3. Pass `0` explicitly when we want read-write mode. This has the problem again that it's not clear at the call site what `0` means.

I would lean a bit towards (1), partly because I tend to think of the ability to write is the interesting property, we always open files in read-mode.

Reply:
Agreed, I'll make that change. Just a moment.